nc2zarr/opener.py

```diff
@@ -163,15 +183,28 @@ def resolve_input_paths(cls,
         resolved_input_files = []
         for input_path in input_paths:
             input_path = os.path.expanduser(input_path)
-            if '*' in input_path or '?' in input_path:
-                glob_result = glob.glob(input_path, recursive=True)
-                if not glob_result:
-                    raise ConverterError(f'No inputs found for wildcard: "{input_path}"')
-                resolved_input_files.extend(glob_result)
+            if 's3' in input_path:
```

Comment:
This needs to be generalized in a central place. It the fourth time in the code this comparison is made.